src/Greeter.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.23;
+
+import {XApp} from "omni/core/src/pkg/XApp.sol";
+import {ConfLevel} from "omni//core/src/libraries/ConfLevel.sol";
+
+import {GreetingBook} from "./GreetingBook.sol";
+
+/**
+ * @title Greeter
+ * @dev This contract is deployed to rollups. It transmits greetings to the GreetingBook on the Omni EVM.
+ */
+contract Greeter is XApp {
+    uint64 public constant DEST_TX_GAS_LIMIT = 120_000;
```

Comment:
Maybe name this something that says that it's for `XCALL_GREET_GAS_LIMIT` or `XGREET_GAS_LIMIT`